backend/Makefile

```diff
@@ -26,6 +26,8 @@ ifeq (${EDNA_USE_CABAL},1)
 		--color $(TEST_ARGUMENTS)
 	call_server = cabal new-run $(PACKAGE):edna-server $(CABAL_BUILD_OPTIONS) -- \
 		$(RUN_ARGUMENTS)
+	call_generator = cabal new-run $(PACKAGE):edna-generator $(CABAL_BUILD_OPTIONS) -- \
+		$(RUN_ARGUMENTS)
```

Comment:
what will happen with `edna-server` if we pass `--projects 4`? As I can see below we call both `edna-server` and `edna-generator` with same arguments (`RUN_ARGUMENTS`), but generator arguments is unknown for `edna-server`

Reply:
`--projects` is parsed only by `edna-generator`, not by `edna-server`. We can have `RUN_GENERATOR_ARGUMENTS` or just `GENERATOR_ARGUMENTS` for `edna-generator`, but I don't see a problem in using `RUN_ARGUMENTS` for both. This way there are less variable names to remember. You run only one of these things, so there should be no ambiguity. Semantic of `RUN_ARGUMENTS` is that they are passed to the executable we want to run, no matter what executable it is.

Reply:
I just tried to call this: `edna-server --projects` and there are an error:
```
Invalid option `--projects'

Usage: edna-server [-c|--config EDNA_CONFIG]
...
```

Reply:
I can try to clone this branch and see what happens if I pass some arguments

Reply:
oh, I can see it now, we do not use generator in tests. Dunno why I thought we use it)

Reply:
> I just tried to call this: `edna-server --projects` and there are an error:

Yes, that's right, as I said, there is no `--projects` option in `edna-server`.
`RUN_ARGUMENTS="--projects 5" make run-generator` makes sense, while `RUN_ARGUMENTS="--projects 5" make run` doesn't. When you type `RUN_ARGUMENTS` you know what command you want to run, so you can write appropriate arguments. So I don't understand the problem. Why would you do `RUN_ARGUMENTS="--projects 5" make run`? You can treat `make run` and `make run-generator` as 2 functions both having an argument call `RUN_ARGUMENTS`. Ideally you would write `make run --run-arguments {arguments}` and `make run-generator --run-arguments {arguments}`, but AFAIU it can't be done easily, so we use env vars instead.

Though if your workflow is to set `RUN_ARGUMENTS` once in your `.bashrc` (or something equvalent) and never touch it, then your suggestion makes sense to me, then you can set `RUN_ARGUMENTS` and `RUN_GENERATOR_ARGUMENTS` to different values and never touch them. Though in this case we should probably add `EDNA` prefix to these variables if you set them globally for you, not just locally for `make`.

Reply:
So what's the use case when it's a problem for you?

Reply:
well, for now it's not a problem.. but if we'll want to use generator along with tests or run some 'stress test' with `make test` it can be a problem..

what do you think?

Reply:
I guess we can resolve this conversation, because:
> oh, I can see it now, we do not use generator in tests. Dunno why I thought we use it)

but we have to come back to this question if
> we'll want to use generator along with tests or run some 'stress test' with `make test`